resources/definitions/fdmprinter.def.json

```diff
@@ -4884,6 +4884,36 @@
                         }
                     }
                 },
+                "build_volume_fan_speed_0":
+                {
+                    "label": "Initial Layers Build Volume Fan Speed",
+                    "description": "Build-volume fan speed until 'Build Volume Fan Speed at Layer'.",
+                    "unit": "%",
+                    "type": "float",
+                    "minimum_value": "0",
+                    "maximum_value": "100",
+                    "default_value": 0,
+                    "value": "0.0",
```

Comment:
The `value` is no required if there is a `default_value` with the same number

Reply:
done!